config.md

```diff
@@ -47,6 +47,7 @@ The parameters are similar to the ones in [the Linux mount system call](http://m
 
 * **`destination`** (string, REQUIRED) Destination of mount point: path inside container.
   For the Windows operating system, one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
+  The destination MUST be present in the root filesystem. Runtime MUST NOT create the destination.
```

Comment:
Also, the "MUST be present in the root filesystem" means that you can't really have nested `tmpfs` mounts anyway. Because `/tmp/a/b` must already have existed in the host root filesystem in order for you to mount it (which would require making the directory). Maybe we should make it something like:

> The destination MUST already be present if the destination mountpoint is on the root filesystem.
> Runtimes MUST NOT modify the container's root filesystem to fulfil destination requirements.